qgis-projects/README.md

```diff
@@ -0,0 +1,141 @@
+# README
+
+This is the readme for a qgis-project data directory.
+
+It has special considerations for the management and workflows of project data due to the requirements for the manual management of conflicts within QGIS project files, as outlined by the relevant [QGIS Project Collaboration SOP Document](https://docs.google.com/document/d/1JkBSQfYDSvlybmNpwsK2DS0egbIRjkiApVBE2B1cGIA/edit?usp=sharing).
+
+## Custodians
+
+The following section outlines data and project custodians for this repository. A custodian is the designated person(s) who should be considered the authoritative source for all modifications to a project or set of source data. All changes should be channeled through and published by the custodian.
+
+---
+
+- Custodian: Default
+- Role: Primary Custodian
+- Description: Responsible for management of project wide geodata and GIS projects
+- Name: Charles Dixon-Paver
+- Mail: charles@kartoza.com
+- GitHub: @zacharlie
+
+---
+
+- Custodian: Lead
+- Role: Project Leader
+- Description: Responsible for decision, especially for making changes to project data structures
+- Name: Ricardo Garcia Silva
+- Mail: ricardo@kartoza.com
+- GitHub: @ricardogsilva
+
+---
+
+## Map Projects
+
+When creating a new map project, please ensure that this section is maintained with the relevant details.
+
+The naming convention for maps and associated files is to use hyphen-separated, or *kebab-case*, styling.
+
+> Map projects should be saved and stored as *qgs* documents (xml) **wherever possible** and not *qgz* documents (zipped archives, which are binary).
+
+The following elements outline the available projects within this repository.
+
+---
+
+* Project: dominode-topomaps.qgs
+* Custodian: Default
+* Description: Primary map project for toposheet production
+
+---
```

Comment:
I think this can be further simplified by leaving most of the descriptive info in the SOP and just stick to the essentials here. Something like:

---
## QGIS Projects

-  {project-name}
   -  custodian: {custodian-email}
   -  {brief-project-description}

---

Also, I don't think the lead custodian role is relevant enough to warrant a description in this file